panel/widgets/__init__.py

```diff
@@ -32,56 +32,86 @@
 For more detail see the Getting Started Guide
 https://panel.holoviz.org/getting_started/index.html
 """
-from .ace import Ace  # noqa
-from .base import Widget, CompositeWidget  # noqa
-from .button import Button, MenuButton, Toggle  # noqa
-from .file_selector import FileSelector  # noqa
-from .indicators import ( # noqa
-    BooleanStatus,
-    Dial,
-    Gauge,
-    LinearGauge,
-    LoadingSpinner,
-    Number,
-    Progress,
-    Trend,
-    Tqdm,
+from .ace import Ace as Ace  # noqa
```

Comment:
*redundant form*. An alternative to `__all__`. 

I don't know which to prefer. Which would you prefer @philippjfr ?

Reply:
Personally I would prefer __all__ because the new developers would ask why add `... as XYZ`, it seems redundant?

But `__all__` might be easier to forget to update?

Reply:
Definitely `__all__`. Is this redundant form really something someone uses? It's crazy!